nominal/cli/auth.py

```diff
@@ -11,6 +14,22 @@ def auth_cmd() -> None:
     pass
 
 
+def _validate_token_url(token: str, base_url: str) -> None:
+    """Ensure the user sets a valid configuration before letting them import the client."""
+    token_link = "https://app.gov.nominal.io/settings/user?tab=tokens"
+    r = requests.get(f"{base_url}/openapi", headers={"Authorization": f"Bearer {token}"})
+    if r.status_code == 401:
+        raise ValueError(
+            f"Your authorization token seems to be incorrect. Please recreate one here: {token_link}"
+        ) from None
```

Comment:
why `raise ... from None`?

Reply:
its just a shorter stacktrace for users to parse through, in case of a nested stacktrace

Reply:
ah gotcha, I ran in a repl and didn't see a difference but that's probably cuz there was no context to suppress!

Reply:
might be better overall to just wrap with a `click.secho(..., error=True)`? only mentioning since we just merged a PR that moves most of the cli printing to use `click.secho`

Reply:
(I don't have a strong opinion though either way here)

Reply:
@drake-nominal should I close this in favor of #88 or should I still update this?

Reply:
updated this